pkg/e2e/Makefile

```diff
@@ -0,0 +1,21 @@
+kubernetes_version=v1.9.0
+
+kube_release_artefacts=https://dl.k8s.io/$(kubernetes_version)/bin/linux/amd64
+
+.PHONY: build
+build: _cache/kubernetes-test.tar.gz _cache/kubectl
```

Comment:
I'd really like to avoid adding any more `Makefile` wrappers for package builds at this point.

Are these things large enough to be worth caching or can we just get away with fetching them in the `Dockerfile`.

If they are big and we think caching is worth while then we should look into enhancing the linuxkit tool to express this in the `build.yml`, perhaps with a new subsection of `depends` alongside the docker image cache one I added to get rid of the last `Makefiles` here? If we are going down this path then probably it would be best to have a design in an issue on the linuxkit/linuxkit repo first.

Note that my preference would be to simply avoid the cache (i.e. just fetch in the `Dockerfile`) unless the artefacts are really so large as to be problematic.